test/unit/client-tests.js

```diff
@@ -55,27 +57,22 @@ describe('Client', function () {
       var Client = rewire('../../lib/client.js');
       var initCounter = 0;
       var emitCounter = 0;
-      var controlConnectionMock = function () {
-        this.hosts = new HostMap();
-        this.metadata = new Metadata();
-        //noinspection JSUnresolvedVariable
-        this.host = { getDistance: utils.noop };
-        this.init = function (cb) {
-          initCounter++;
-          //Async
-          setTimeout(cb, 100);
-        }
-      };
-      Client.__set__("ControlConnection", controlConnectionMock);
       var options = utils.extend({
         contactPoints: helper.baseOptions.contactPoints,
         policies: {
           loadBalancing: new policies.loadBalancing.RoundRobinPolicy()
         }
       });
+      var ccMock = getControlConnectionMock(null, options);
+      ccMock.prototype.init = function (cb) {
+        initCounter++;
+        //Async
+        setTimeout(cb, 100);
+      };
+      Client.__set__("ControlConnection", ccMock);
       var client = new Client(options);
       client.on('connected', function () {emitCounter++;});
-      utils.times(1000, function (n, next) {
+      utils.timesSeries(1, function (n, next) {
```

Comment:
is this accidental?  It still seems to work with `util.times(100...)` and appears to be the intent of the test (to try multiple times and ensure init'd once)